src/components/button/button.tsx

```diff
@@ -19,55 +19,154 @@ export class Button {
       <ion-content fullscreen={true}>
         <section>
           <ion-list-header>
-            <ion-label>
-              Small Size
-            </ion-label>
+            <ion-label>Colors</ion-label>
           </ion-list-header>
-          <ion-button size="small">Default</ion-button>
-          <ion-button size="small" color="secondary">Secondary</ion-button>
-          <ion-button size="small" color="tertiary">Tertiary</ion-button>
+          <div class="content">
+            <ion-button color="success">Success</ion-button>
+            <ion-button color="warning">Warning</ion-button>
+            <ion-button color="danger">Danger</ion-button>
+            <ion-button color="secondary">Secondary</ion-button>
+            <ion-button color="tertiary">Tertiary</ion-button>
+            <ion-button color="light">Light</ion-button>
+            <ion-button color="medium">Medium</ion-button>
+            <ion-button color="dark">Dark</ion-button>
+          </div>
         </section>
 
         <section>
           <ion-list-header>
-            <ion-label>
-              Default Size
-            </ion-label>
+            <ion-label>Sizes</ion-label>
           </ion-list-header>
-          <ion-button color="success">Success</ion-button>
-          <ion-button color="warning">Warning</ion-button>
-          <ion-button color="danger">Danger</ion-button>
+          <div class="content">
+            <ion-button size="small">Small</ion-button>
+            <ion-button>Default</ion-button>
+            <ion-button size="large">Large</ion-button>
+            <ion-button size="xlarge">XLarge</ion-button>
```

Comment:
Should we have some kind of message that the `xlarge` size is only available in the `ionic` theme, or maybe only render it if you have the theme turned on the in the app? Currently it looks like a bug when viewed in any other theme.